test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt

```diff
@@ -157,8 +157,11 @@ com/sun/crypto/provider/TLS/TestPRF.java https://github.com/eclipse-openj9/openj
 com/sun/crypto/provider/TLS/TestPRF12.java https://github.com/eclipse-openj9/openj9/issues/20978 generic-all
 com/sun/jdi/JdwpAttachTest.java.JdwpAttachTest https://github.com/eclipse-openj9/openj9/issues/20978 generic-all
 com/sun/jndi/dns/ConfigTests/PortUnreachable.java https://github.com/eclipse-openj9/openj9/issues/20978 generic-all
+com/sun/jndi/ldap/DeadSSLLdapTimeoutTest.java https://github.com/eclipse-openj9/openj9/issues/20978 aix-all,linux-ppc64le,linux-s390x,linux-x64
+com/sun/jndi/ldap/DeadSSLLdapTimeoutTest.java https://github.com/eclipse-openj9/openj9/issues/20978 generic-all
```

Comment:
Please remove duplications like this: `generic-all` includes all the platforms listed on the previous line.